PurchasesCoreSwift/FoundationExtensions/String+Extensions.swift

```diff
@@ -23,6 +23,12 @@ extension String {
 
 }
 
+extension Optional where Wrapped == String {
+
+    func rc_orEmpty() -> String { return self ?? "" }
```

Comment:
Maybe `rc_valueOrEmpty` is a better name?

Reply:
sure, i got .orEmpty from kotlin, but it's not the most descriptive. actually now that it has the rc prefix it doesn't really save characters... do you still think it's worth keeping around?

Reply:
Ah, good point. Probably not worth keeping it around.